packages/configurationmap/components/ConfigurationMap.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+// import { style } from "../style";
+import { Table, Column } from "../../";
+
+export interface ConfigurationMapProps {
+  data: { [key: string]: string };
+}
+
+const cellRenderer = column => data => <span>{data[column]}</span>;
+
+class ConfigurationMap extends React.PureComponent<ConfigurationMapProps, {}> {
+  public render() {
+    const { data } = this.props;
+
+    const tableData = Object.entries(data);
+
+    return (
+      <Table data={tableData}>
```

Comment:
Do you think we'll use the `Table` component for the real ConfigurationMap implementation? I'd prefer we didn't - it carries a lot of JavaScript to execute, and we'd have to come up with a way to override some of the styles (e.g.: the row hover background color change)

Reply:
I don't think that we are going to use the table in the final implementation. My thinking was like a configuration map is a two column table which keeps key's and values. And the way of the least effort was to just use the table for it. Alternative could have been to use html table. And I agree that the table is not the best implementation for the final configuration map. Although the row highlight could be helpful ;-)

Reply:
I would vote for using the default html table.